### PR TITLE
Shaking hand

### PR DESCRIPTION
**Pull request by Pratham-Mishra178**
Hi have added a shaky hand instead of the static one in the home page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -87,7 +87,7 @@
     </div>
 <center>
     <!-- Home Section -->
-    <div class="text col-sm-6">Hi <span class="wave">👋</span>,I'm <span class="header">John Doe</span>
+    <div class="text col-sm-6">Hi <span class="wave"><img src="https://raw.githubusercontent.com/ABSphreak/ABSphreak/master/gifs/Hi.gif" width="35px"></span>&nbsp;,I'm <span class="header">John Doe</span>
```
Comment (smaranjitghose) [inline index.html:90]: The assets should be present locally

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Pratham-Mishra178) [reply index.html:90]: okay

Comment (Pratham-Mishra178) [reply index.html]: content put in the assets.

**Comment by smaranjitghose** [comment]
respon
 Responsiveness Issue for screen sizes similar to Pixel 2 XL

**Comment by Pratham-Mishra178** [comment]
> respon
> Responsiveness Issue for screen sizes similar to Pixel 2 XL

ok resolving it

**Comment by Pratham-Mishra178** [comment]
Screenshot (504)

But it looks great at my place

**Comment by Pratham-Mishra178** [comment]
Now its looking great, Check it

**Review by smaranjitghose — APPROVED** [review]

**Pull request merged**